start/sw/web-app/laravel5.md

```diff
@@ -58,10 +58,10 @@ $ composer -V
 $ composer global require "laravel/installer"
 # Place the ~/.config/composer/vendor/bin directory in your PATH so the laravel 
 # executable can be located by your system.
-$ echo 'export PATH="$PATH:$HOME/.config/composer/vendor/bin"' >> ~/.bash_profile
```

Comment:
@pvalena could you please take a look at the [stackoverflow](http://stackoverflow.com/questions/26376516/laravel-php-command-not-found) issue?